src/Engine/Utils.java

```diff
@@ -319,4 +320,35 @@ public int compare(SearchNode o1, SearchNode o2)
       return firstPow - secondPow;
     }
   }
+
+  /**
+   * Returns a list of all vacant industries a commander owns
+   */
+  public static ArrayList<XYCoord> findFreeIndustries(Commander co, GameMap map)
+  {
+    ArrayList<XYCoord> industries = new ArrayList<XYCoord>();
+    // we don't want to bother if we're trying to find nobody's properties :v
+    if( null != co )
+    {
+      // Add all vacant, <co>-owned industries to the list
+      for( int i = 0; i < map.mapWidth; i++ )
```

Comment:
Ooh. I don't really like iterating over the entire map here, especially since you do it every time the `SEEK` button is hit. If you look at the `resupply` branch, I've changed all or most of the `Utils` functions to not do that.
I don't think we are super worried about performance, but it might be worth giving the commander a list of the property locations it owns; that would let us just iterate over that list here, and would also be useful for abilities that depend on/use owned properties.